mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -13,8 +13,7 @@
                         </div>
                         <div class="oe_mis_builder_cp_right_bottom">
                             <div class="oe_mis_builder_filter_buttons">
-                                <FilterMenu t-if="showSearchBar" />
```

Comment:
Is it ok to remove this (I have not investigated yet).

Reply:
<SearchBar> does already contain menu in 17.